app/styles/app.sass

```diff
@@ -2120,7 +2118,13 @@ button.delete-button
     height: 50px
     left: 52%
 
+.novnc-rfb
```

Comment:
move this to local class

Reply:
`novnc-rfb` is used both at the component (novnc-rfb) level class and the child div inside the parent component (novnc-rfb) and both needs width and height `100%`, so I move this to local class then I am not getting how to bind the local class with the component in ember

Reply:
Netlify is failing with the log, `URL not defined`, I am building the app locally and it's building without any errors.